### ISSUE TITLE
Allow middle click to open new tab

### ISSUE THREAD

**Issue by amenk** [stevehanson/domain-switcher-chrome#9]
When clicking a domain in the domain switcher with the third mouse button, a new tab should be opened

**Comment by stevehanson** [comment]
Good call. Feel free to open a PR, or I'll try to get around to it at some point.

**Comment by amenk** [comment]
Does not work anymore in Chrome 55. Workaround seems to be to set also the href attribute of the item in the popup. But for getting the actual href, some restructuring would be necessary. Not even sure if it is possible.
Also I was not able to use `href="javascript: ...."` because of CSP
http://www.cspplayground.com/compliant_examples#javascript-uri
